src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/WebScenarioTests.cs

```diff
@@ -37,7 +37,8 @@ private static IEnumerable<TestScenario> GetScenarios()
 
         yield return new(nameof(WebScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.Razor,         DotNetActions.Build | DotNetActions.Run | DotNetActions.Publish);
         yield return new(nameof(WebScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.BlazorWasm,    DotNetActions.Build | DotNetActions.Run | DotNetActions.Publish);
-        yield return new(nameof(WebScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.WebApp,        DotNetActions.PublishSelfContained, VerifyRuntimePacksForSelfContained);
+        // Disabled due to .NET 10.0 transition. See https://github.com/dotnet/sdk/pull/42969
```

Comment:
please file a tracking issue for this

Reply:
https://github.com/dotnet/source-build/issues/4578